### PR TITLE
MULE-12688: Make WSC DW and MediaType friendly

### PR DESCRIPTION
**Pull request by estebanwasinger**
and move IT's to connector's repo

### CODE REVIEW

File: src/main/java/org/mule/extension/ws/api/SoapMessageBuilder.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
   @Content(primary = true)
   @TypeResolver(SoapBodyTypeResolver.class)
   @Summary("The XML body to include in the SOAP message, with all the required parameters.")
-  private InputStream body;
+  private TypedValue<InputStream> body;
```
Comment (juandesi) [inline src/main/java/org/mule/extension/ws/api/SoapMessageBuilder.java]: this should not be a typed value

Comment (marianogonzalez) [inline src/main/java/org/mule/extension/ws/api/SoapMessageBuilder.java]: why not?

File: src/main/java/org/mule/extension/ws/internal/ConsumeOperation.java [hunk 2]
```diff
@@ -46,9 +53,13 @@
  */
 public class ConsumeOperation {
 
+  private static final DataType XML_STREAM = DataType.builder().type(InputStream.class).mediaType(XML).build();
   @Inject
   private MuleExpressionLanguage expressionExecutor;
 
+  @Inject
+  TransformationService transformationService;
```
Comment (marianogonzalez) [inline src/main/java/org/mule/extension/ws/internal/ConsumeOperation.java]: private

File: src/main/java/org/mule/extension/ws/api/SoapMessageBuilder.java (hunk 1 above)
Comment (juandesi) [inline src/main/java/org/mule/extension/ws/api/SoapMessageBuilder.java]: always xml

**Pull request merged**